### PR TITLE
refactor: Migrate GroupVideoGrid to React

### PR DESCRIPTION
**Pull request by thisisamir98**

### CODE REVIEW

File: src/script/components/calling/GroupVideoGrid.tsx [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+import {t} from '../../util/LocalizerUtil';
+
+export interface GroupVideoGripProps {
+  grid: Grid;
+  minimized?: boolean;
+  muted?: boolean;
+  selfParticipant: Participant;
+}
+
+interface RowsAndColumns {
```
Comment (Yserz) [inline src/script/components/calling/GroupVideoGrid.tsx]: Does `extends CSSProperties` help here?

File: src/script/components/calling/GroupVideoGrid.tsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+}
+
+const calculateRowsAndColumns = (totalCount: number): RowsAndColumns => {
+  const columns = Math.ceil(Math.sqrt(totalCount));
+  const rows = Math.ceil(totalCount / columns);
+  return {'--columns': columns, '--rows': rows};
+};
+
+const GroupVideoGrid: React.FunctionComponent<GroupVideoGripProps> = ({
+  minimized,
```
Comment (Yserz) [inline src/script/components/calling/GroupVideoGrid.tsx]: Does it make sense to make this `minimized = false`?

File: src/script/components/calling/GroupVideoGrid.tsx (hunk 1 above)
Comment (thisisamir98) [reply src/script/components/calling/GroupVideoGrid.tsx]: https://github.com/wireapp/wire-webapp/pull/10025/commits/a3060e643ffa3d1a0238e13add6d7189d71f40e0

File: src/script/components/calling/GroupVideoGrid.tsx (hunk 2 above)
Comment (thisisamir98) [reply src/script/components/calling/GroupVideoGrid.tsx]: https://github.com/wireapp/wire-webapp/pull/10025/commits/646a1dccf19bdba34f3f7e883bfa4ae8e248e17b

**Review by lipis — APPROVED** [review]
Who said react is better?! `+363` and `−151` 😛

**Review by Yserz — APPROVED** [review]

**Pull request merged**